src/js/components/Article.js

```diff
@@ -439,19 +440,29 @@ export default class Article extends Component {
     if ('row' === this.props.direction) {
       if (! this.state.narrow || this.state.atBottom) {
         if (this.state.selectedIndex > 0) {
+          let controlSpacer = (!this.state.narrow) ? (<Paragraph size="large"> &nbsp; </Paragraph>) : null; 
           controls.push(
-            <Button key="previous" ref='previous'
-              plain={true} a11yTitle={a11yTitle.previous}
-              className={`${CONTROL_CLASS_PREFIX}-left`}
-              onClick={this._onPrevious} icon={<PreviousIcon size="large" />} />
+            <Box key="previous" ref='previous' className={`${CONTROL_CLASS_PREFIX}-left`} direction="row"
+            onClick={this._onPrevious} pad={{vertical: "small", horizontal: "medium"}}>
```

Comment:
i think there is a spacing issue here `onClick` needs to be shifted to characters to the right